prepare/metrics/customf1fuzzy.py

```diff
@@ -0,0 +1,359 @@
+from unitxt import add_to_catalog
+from unitxt.metrics import FuzzyNer
+from unitxt.test_utils.metrics import test_metric
+
+metric = FuzzyNer()
+# Test1 single line single class
+# 0.1 simple case, multi examples
+predictions = [
+    [("Amir", "Person"), ("Yaron", "Person")],
+    [("Ran", "Person"), ("Yonatan", "Person")],
+]
+references = [[[("Yaron", "Person"), ("Ran", "Person")]], [[("Yonatan", "Person")]]]
```

Comment:
I think we need to modify the tests to see that there Fuzzy matching makes a difference compared to regular NER.

Reply:
we will update different data to check on the values